### DISCUSSION TITLE
Bad Gateway

### DISCUSSION THREAD

**Discussion by serdarakyol** [deta/cloud-docs#709] [Help] [answered]
I deployed my project.  My endpoint is `https://9dep76.deta.dev`  and I wanted to go to the `https://9dep76.deta.dev/docs`  but I got the following error: 
```
Bad Gateway
The micro ran into an error while processing the request.

If you are the owner of the micro, please check your logs.
```
[FastAPI docs](https://fastapi.tiangolo.com/deployment/deta/#check-it) shows that I can use the swagger UI. Is that possible or not?

**Comment by lemonyte** [comment, marked as answer]
This is likely not an issue with FastAPI or Swagger, please check the Micro's logs by running the `deta logs` command to see why you got a Bad Gateway error.
